backend/spec/controllers/spree/admin/return_items_controller_spec.rb

```diff
@@ -34,7 +34,13 @@
       let(:new_reception_status) { 'received' }
       let(:reception_status_event) { 'receive' }
 
-      before { return_item.update reception_status: 'in_transit'}
+      before do
+        allow(Spree::Deprecation).to receive(:warn) do |message|
+          expect(message).to match('#process_inventory_unit! will not call')
+        end
+
+        return_item.update reception_status: 'in_transit'
```

Comment:
Usually I use `update!` when setting up a test. That way, if the setup code fails for some reason, it does so loudly instead of making the expectations fail.